### ISSUE TITLE
Release in Google Play

### ISSUE THREAD

**Issue by gargullia** [Gwokhov/Deadline#5]
Hello. Will the application release in Google Play?
If not, can you make a fork with your add-ons and put it in this store?

**Comment by gwokhov** [comment]
Google Play is not in consideration temporarily.
> If not, can you make a fork with your add-ons and put it in this store?

Add-ons? You mean the dependencies?

**Comment by gargullia** [comment]
Not. I mean, new features and functionality. As well as design.

**Comment by gwokhov** [comment]
It may not be updated in the short term.

**Comment by gargullia** [comment]
But the license allows you to create fork and various software modifications.  You can change the name of the application package so that it does not conflict with your version.

**Comment by gwokhov** [comment]
Yes, you're right. But now I may primarily focus on other projects, so it's not so sure when I will update it.

**Comment by gargullia** [comment]
Then I will fork. I will transfer to Kotlin and update functions. Is it possible to source the icons to put in Google?

**Comment by gargullia** [comment]
Need files ic_launcher_foreground and ic_launcher_backgroung in xml or svg

**Comment by gwokhov** [comment]
I think It's ok. Make sure it is following GPL license and provide attribution.

**Comment by gargullia** [comment]
In Google Play store can block for images from the stock. If you are not the author. So if you're an author, add vector graphics.
